lib/bmp

```diff
@@ -15,16 +15,12 @@ u32le() {
 	# octet2: 0x34
 	# octet3: 0x56
 	# octet4: 0x78
-	local octet1=$(( (n >> 24) & 0xFF))
-	local octet2=$(( (n >> 16) & 0xFF))
-	local octet3=$(( (n >> 8)  & 0xFF))
-	local octet4=$(( (n >> 0)  & 0xFF))
-
-	printf -v out '\\x%02x\\x%02x\\x%02x\\x%02x' \
-	    "$octet4" \
-	    "$octet3" \
-	    "$octet2" \
-	    "$octet1"
+	local octet1=$(( (n >> 0)  & 0xFF))
```

Comment:
changing this means tho comment above no longer aligns with the data being stored - i purposefully parsed them in this order to show that "little-endian" is backwards to how a left-to-right language is written, so i prefer leaving this as-is.